An introduction for the codeschool assignment

guillaumemaiano opened the pull request:
A _presentation_ inside a **presentation file**. I'm so **_amazing**_.

| Option | Option |
| --- | --- |
| YES | NO |

PeterBell committed:
Merge pull request #2 from githubteacher/master

Most Senseis should be respected

Commit:
Initial kill list

PeterBell committed:
Merge pull request #3 from deadlyvipers/kill_list

Initial kill list

Commit:
We dont need to kill Budd

Commit:
An introduction for the codeschool assignment

codeschool-kiddo commented on introduction.md:
```diff
@@ -0,0 +1 @@
+Hello, I'm a monkey!
```
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

Commit:
An introduction for the codeschool assignment, with a twist

Commit:
Added useful deadly skills

Commit:
Added Markup to the markdown file

guillaumemaiano committed:
Update introduction.md

guillaumemaiano committed:
Merge pull request #1 from guillaumemaiano/deadly_skills

Deadly skills

Commit:
Messed it up
Merge branch 'master' of github.com:guillaumemaiano/dojo_rules into release_branch_1.0

Commit:
added changes again

Commit:
dop dop

Commit:
Merge branch 'release_branch_1.0'

Commit:
useless file

Commit:
Cleaner, and fixes #3
